pcdsdevices/utils.py

```diff
@@ -160,8 +160,11 @@ def ipm_screen(dettype, prefix, prefix_ioc):
     if shutil.which(executable) is None:
         raise EnvironmentError('%s is not on path, we cannot start the screen'
                                % executable)
-    os.system('%s --base %s --ioc %s --evr %s &' %
-              (executable, prefix, prefix_ioc, prefix+':TRIG'))
+
+    logger.info(f'Opening {dettype} screen for {prefix}...')
+    arglist = [executable, '--base', prefix, '--ioc', prefix_ioc,
+               '--evr', prefix+':TRIG']
+    _ = subprocess.Popen(arglist)
```

Comment:
What's the functional difference between binding the Popen output to an unused variable vs no assignment at all?

Reply:
I don't entirely understand why, but in this case `Popen` was returning a value that printed a repr to the terminal.  Each external script behaves a little differently, but I didn't spend a lot of time digging into why.